go/Hash Maps and Sets/pair_sum_unsorted.go

```diff
@@ -0,0 +1,11 @@
+func pairSumUnsorted(nums []int, target int) []int {
+	hashmap := make(map[int]int)
+	for i, x := range nums {
+		complement := target - x
+		if idx, exists := hashmap[complement]; exists {
+			return []int{idx, i}
+		}
+		hashmap[x] = i
+	}
+	return []int{}
```

Comment:
Apologies. I'm removing this suggestion per our chat in Discord: Returning `nil` is a common and idiomatic way in Go to indicate the absence of a result, as it differentiates between "no result" and "an empty result."

We should keep 'nil' here.